translations/ja.json

```
@@ -761,55 +761,55 @@
+	"Upgrade to QEMU v3.1.0+, run 'virt-host-validate', or ensure that you are not running in a nested VM environment.": "QEMU v3.1.0 以降にアップグレードするか、'virt-host-validate' を実行するか、ネストされた VM 環境中で実行されていない事を確認してください。",
 	"Upgrading from Kubernetes {{.old}} to {{.new}}": "Kubernetes を {{.old}} から {{.new}} にアップグレードしています",
-	"Usage": "",
-	"Usage: minikube completion SHELL": "",
-	"Usage: minikube delete": "",
-	"Usage: minikube delete --all --purge": "",
-	"Usage: minikube node [add|start|stop|delete|list]": "",
-	"Usage: minikube node delete [name]": "",
-	"Usage: minikube node list": "",
-	"Usage: minikube node start [name]": "",
-	"Usage: minikube node stop [name]": "",
-	"Use \"{{.CommandPath}} [command] --help\" for more information about a command.": "",
-	"Use 'kubect get po -A' to find the correct and namespace name": "",
-	"Use -A to specify all namespaces": "",
-	"Use SSH connection instead of HTTPS (port 2376)": "",
-	"Use SSH for running kubernetes client on the node": "",
-	"Use VirtualBox to remove the conflicting VM and/or network interfaces": "",
-	"Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'.": "",
+	"Usage": "使用法",
+	"Usage: minikube completion SHELL": "使用法: minikube completion SHELL",
+	"Usage: minikube delete": "使用法: minikube delete",
+	"Usage: minikube delete --all --purge": "使用法: minikube delete --all --purge",
+	"Usage: minikube node [add|start|stop|delete|list]": "使用法: minikube node [add|start|stop|delete|list]",
+	"Usage: minikube node delete [name]": "使用法: minikube node delete [ノード名]",
+	"Usage: minikube node list": "使用法: minikube node list",
+	"Usage: minikube node start [name]": "使用法: minikube node start [ノード名]",
+	"Usage: minikube node stop [name]": "使用法: minikube node stop [ノード名]",
+	"Use \"{{.CommandPath}} [command] --help\" for more information about a command.": "コマンドに関する追加情報は「{{.CommandPath}} [command] --help」を使用してください。",
+	"Use 'kubect get po -A' to find the correct and namespace name": "'kubect get po -A' を使用して、妥当なネームスペース名を見つけてください",
+	"Use -A to specify all namespaces": "-A を使用して、全ネームスペースを指定してください",
+	"Use SSH connection instead of HTTPS (port 2376)": "HTTPS 接続の代わりに SSH 接続を使用してください (ポート 2376)",
+	"Use SSH for running kubernetes client on the node": " SSH を使用して、ノード上の Kubernetes クライアントを実行ください",
+	"Use VirtualBox to remove the conflicting VM and/or network interfaces": "VirtualBox を使用して、衝突した VM やネットワークインターフェイスを削除してください",
+	"Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'.": "ネイティブの Go 言語 SSH クライアントを使用してください (デフォルトは true)。Docker マシンにアクセスする際には、'false' をセットして、コマンドラインの 'ssh' コマンドを使用してください。マシンドライバーが 'Waiting for SSH' で開始されない場合に有用です。",
```

`--native-ssh` option on [here](https://minikube.sigs.k8s.io/docs/commands/ssh/#options) is the place?
If so, it's better I think.
 ```suggestion
	"Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'.": "ネイティブの Go 言語 SSH クライアントを使用します (デフォルトは true)。Docker マシンにアクセスする際に、コマンドラインの 'ssh' コマンドを使用する場合は 'false' をセットしてください。マシンドライバーが 'Waiting for SSH' で開始されない場合に有用です。",
```

---

Oh, it does not looks the exact text. but meaning is same with this text, right?